cohorts/cohort.py

```diff
@@ -369,6 +377,7 @@ def load_dataframe(self, df_loader_name):
         Instead of joining a DataFrameJoiner with the Cohort in `as_dataframe`, sometimes
         we may want to just directly load a particular DataFrame.
         """
+        logger.debug('loading dataframe: {}'.format(df_loader_name))
```

Comment:
I tried to find most of these -- i have the opposite convention but will try to adjust